python/py-jupyterlab/Portfile

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim: fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           python 1.0
+
+name                py-jupyterlab
+version             0.33.12
+revision            0
+categories-append   devel
+platforms           darwin
+license             BSD
+supported_archs     noarch
+
+python.versions     27 34 35 36
+
+maintainers         {stromnov @stromnov} openmaintainer
+
+description         Next-generation web-based notebook environment for \
+                    interactive computing.
+long_description    JupyterLab enables you to work with documents and \
+                    activities such as Jupyter notebooks, text editors, \
+                    terminals, and custom components in a flexible, \
+                    integrated, and extensible manner.
+
+homepage            http://jupyter.org
+master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
+
+distname            ${python.rootname}-${version}
+
+checksums           rmd160  4fe0074a1202cbc3833d1a85a3b8b9c5c97701c2 \
+                    sha256  04a14b38b218c87731b519ff0f7e133b28a613ad944617ac7e07480d6a36eb5f
```

Comment:
These days we would traditionally also have a "size" line as part of "checksums".